packages/driver/src/cy/commands/navigation.coffee

```diff
@@ -308,16 +308,14 @@ module.exports = (Commands, Cypress, cy, state, config) ->
         when not resp.isOkStatusCode
```

Comment:
Can you do `git checkout develop -- packages/driver/src/cy/commands/navigation.coffee` and revert all the changes to this file? Looks like just leftover whitespace changes